src/main/java/org/opentripplanner/transit/raptor/rangeraptor/transit/TransitCalculator.java

```diff
@@ -130,7 +129,10 @@ static <T extends RaptorTripSchedule> TransitCalculator<T> testDummyCalculator(b
    * target} is the TO pattern/stop, while when searching in reverse the given target is the FROM
    * pattern/stop.
    */
-  RaptorConstrainedTripScheduleBoardingSearch<T> transferConstraintsSearch(RaptorRoute<T> route);
+  RaptorConstrainedTripScheduleBoardingSearch<T> transferConstraintsSearch(
+    RaptorTransitDataProvider<T> transitData,
```

Comment:
It would be possible, but would complicate som tests, where we would need to create a `RaptorTransitDataProvider<T>`. There is a similar issue with `getTransfers`